src/uucore/src/lib/features/fs.rs

```diff
@@ -648,6 +648,20 @@ pub fn are_hardlinks_to_same_file(source: &Path, target: &Path) -> bool {
     source_metadata.ino() == target_metadata.ino() && source_metadata.dev() == target_metadata.dev()
 }
 
+pub fn are_hardlinks_or_oneway_symlink_to_same_file(source: &Path, target: &Path) -> bool {
```

Comment:
please document this function so that we get the same as
https://uutils.github.io/dev/uucore/fs/fn.are_hardlinks_to_same_file.html